src/components/GhostContracts.vue

```diff
@@ -1,82 +1,118 @@
 <template>
-  <section>
-    <h2>Contracts</h2>
-    <div class="contracts">
-      <div
-        v-for="contract in contracts"
-        :key="contract.id"
-        @click="openContractDetails(contract)"
-        class="contract"
-      >
-        <h3>{{ contract.location.name }}</h3>
-        <p>Size: {{ contract.location.size }}</p>
-      </div>
-    </div>
-    <div
-      class="contract-modal"
-      v-if="contractDetails"
-      @click.self="contractDetails = null"
-    >
-      <div class="contract-details">
-        <div class="header">
-          <h1>{{ contractDetails.location.name }}</h1>
-          <p>Size: {{ contractDetails.location.size }}</p>
-          <p>Difficulty: Amateur</p>
-          <p>Reward: ${{ contractDetails.location.baseReward }}</p>
-          <br />
-        </div>
-        <div class="body">
-          <h2>Assign team</h2>
-          <div class="team-picker">
-            <div
-              v-for="hunter in chosenTeam"
-              :key="hunter.id"
-              @click="removeHunterFromChosenTeam(hunter)"
-              class="team-picker__spot"
-            >
-              <p>{{ hunter.name }}</p>
-            </div>
-            <div
-              class="team-picker__spot"
-              @click="showAvailableHunters = !showAvailableHunters"
-            >
-              +
+  <b-container class="py-2" fluid>
+    <b-row align-h="center" align-v="center">
+      <b-col>
+        <div class="contracts p-4">
+          <h2 v-b-modal.modal-contracts>Contracts</h2>
+          <b-container cols="12">
+            <b-row align-h="between" align-v="center">
+              <b-col
+                v-b-modal.contractmodal
+                cols="12"
+                v-for="contract in contracts"
+                :key="contract.id"
+                @click="openContractDetails(contract)"
+                class="contract-btn p-3 my-1"
+              >
+                <h3>{{ contract.location.name }}</h3>
+                <p>Size: {{ contract.location.size }}</p>
+              </b-col>
+            </b-row>
+          </b-container>
+          <b-modal
```

Comment:
Er der en Modal template/component vi kan bruge her i stedet for at det står statisk?